### PR TITLE
Mikufan.com

### PR DESCRIPTION
**Pull request by Stephhawks**
Canceled the header image inversion

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -8947,6 +8947,15 @@ body,
 
 ================================
 
+mikufan.com
+
+CSS
+#header-container {
+    background-image: url(https://www.mikufan.com/wp-content/uploads/2018/04/header_main3.jpg) !important;
+}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:8955]:
```suggestion
IGNORE IMAGE ANALYSIS
#header-container
```

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (Stephhawks) [reply src/config/dynamic-theme-fixes.config:8955]: It didn't work when i tried

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:8955]: You've to reload the page before it takes effect.

Comment (Stephhawks) [reply src/config/dynamic-theme-fixes.config]: I see, thanks

**Review by Gusted — APPROVED** [review]
Thanks!

**Pull request merged**